src/components/StoryBoard.vue

```diff
@@ -116,8 +137,9 @@
                 });
 
                 // add the layers we changed to the component data, so that the next time the toggle is run we can reset them
-                this.layersToUnhide = layersToHide;
-                this.layersToUnshow = layersToShow;
+                self.layersToUnhide = layersToHide;
```

Comment:
these var names are causing me to 🙃  -- any reason these are named oppositely?

Reply:
Yes. The names are relative to the state in which the map is loaded. So when the map is loaded if the layer is 'visibility: none', I consider it hidden. So you can choose to 'unhide' it. The opposite is true for the layers to show. I am open to better solutions, but I thought this helped keep it clear that the selections were related to the on load state of the map rather than the current state.